src/test/ui/imports/issue-62767.rs

```diff
@@ -20,9 +22,9 @@ mod foo {
     }
 }
 
+use bar::bar;
 use foo::*;
-use bar::bar; //~ ERROR unresolved import `bar::bar`
-              //~| ERROR inconsistent resolution for an import
```

Comment:
IIRC, the import order was important for reproducing the ICE.

Reply:
Good catch, I was editing in a different directory and didn't notice rustfmt reformatting the file. Adjusted back.

Reply:
(Is there a way to tell rustfmt not to do this for this file, via an embedded comment or attribute in the source?)

Reply:
It's not going to do it via x.py fmt, but I was editing in a different workspace and hadn't copied my vim config to it.